### PR TITLE
Add SL Editor v1.4

### PR DESCRIPTION
**Pull request by JustJarethB**
**Important:** *Do not tick a checkbox if you haven’t performed its action.* Honesty is indispensable for a smooth review process.

_In the following questions `<cask>` is the token of the cask you're submitting._

After making all changes to a cask, verify:

- [x] The submission is for [a stable version](https://docs.brew.sh/Acceptable-Casks#stable-versions) or [documented exception](https://docs.brew.sh/Acceptable-Casks#but-there-is-no-stable-version).
- [x] `brew audit --cask --online <cask>` is error-free.
- [x] `brew style --fix <cask>` reports no offenses.

Additionally, **if adding a new cask**:

- [x] Named the cask according to the [token reference](https://docs.brew.sh/Cask-Cookbook#token-reference).
- [x] Checked the cask was not [already refused](https://github.com/Homebrew/homebrew-cask/search?q=is%3Aclosed&type=Issues).
- [x] Checked the cask is submitted to [the correct repo](https://docs.brew.sh/Acceptable-Casks#finding-a-home-for-your-cask).
- [x] `brew audit --new-cask <cask>` worked successfully.
- [x] `brew install --cask <cask>` worked successfully.
- [x] `brew uninstall --cask <cask>` worked successfully.

### CODE REVIEW

File: Casks/sl-editor.rb [hunk 1]
```diff
@@ -0,0 +1,21 @@
+cask "sl-editor" do
+  version "1.4"
+  sha256 "38dd6fa8aead4fcd11c9d26d59bd7a5845c6fe769e907159677e27cadd9ae2fc"
+
+  url "https://www.studiologic-music.com/support/articles/sl_editor/SL_Editor_#{version}.pkg"
+  name "SL Editor"
+  desc "Editor for StudioLogic Devices"
```
Comment (neutric) [inline Casks/sl-editor.rb:7]: Is this software only useful for StudioLogic products? If so, this probably needs to be submitted to the [`homebrew-cask-drivers` repository](https://github.com/Homebrew/homebrew-cask-drivers).

File: Casks/sl-editor.rb [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+  homepage "https://www.studiologic-music.com/"
```
Comment (neutric) [inline Casks/sl-editor.rb:8]: `homepage` should point to the software this cask is about. Is there anything better than https://www.studiologic-music.com/support/articles/sl_editor/ ?

File: Casks/sl-editor.rb [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+
+  livecheck do
+    url "https://www.studiologic-music.com/support/articles/sl_editor/"
+    strategy :page_match
```
Comment (neutric) [inline Casks/sl-editor.rb:12]: `strategy :page_match` should only be used when explicitly needed. It is the default except for special cases such as GitHub-URLs.

File: Casks/sl-editor.rb [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+    regex(/href=.*?SL_Editor_(\d+(?:\.\d+)*)\.pkg/i)
```
Comment (neutric) [inline Casks/sl-editor.rb:13]:
```suggestion
    regex(/href=.*?SL_Editor[._-]v?(\d+(?:\.\d+)+)\.pkg/i)
```

File: Casks/sl-editor.rb [hunk 5]
```diff
@@ -0,0 +1,21 @@
...
+  end
+
+  pkg "SL_Editor_#{version}.pkg"
+
+  uninstall pkgutil: [
+    "com.fatar.studiologic.pkg.SLEditor",
+  ]
```
Comment (neutric) [inline Casks/sl-editor.rb:20]: Please try to add a [`zap` stanza](https://docs.brew.sh/Cask-Cookbook#stanza-zap). The linked documentation provides some recommended methods to create it.

**Review by neutric — COMMENTED** [review]
Thanks for your contribution to `homebrew-cask`, @JustJarethB! My review is in the code. Take it with a grain of salt as I am not a maintainer.

**Comment by miccal** [comment]
Thank you @JustJarethB, but as pointed out by @neutric please submit this to the [`homebrew-cask-drivers` repository](https://github.com/Homebrew/homebrew-cask-drivers).

**Pull request closed without merging**